topdown/crypto.go

```diff
@@ -7,9 +7,10 @@ package topdown
 import (
 	"crypto/x509"
 	"encoding/json"
-
+	"fmt"
 	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/util"
+	"io/ioutil"
 )
```

Comment:
Let's sort these a bit:
```go
import (
	"crypto/x509"
	"encoding/json"
	"fmt"
	"io/ioutil"

	"github.com/open-policy-agent/opa/ast"
	"github.com/open-policy-agent/opa/util"
)
```
I believe `goimports -local github.com/open-policy-agent/opa` would achieve the same.